CHANGELOG.next.asciidoc

```diff
@@ -143,6 +143,7 @@ https://github.com/elastic/beats/compare/v8.8.1\...main[Check the HEAD diff]
 - Fix metric collection in GCPPubSub input. {pull}35773[35773]
 - Fix end point deregistration in http_endpoint input. {issue}35899[35899] {pull}35903[35903]
 - Fix duplicate ID panic in filestream metrics. {issue}35964[35964] {pull}35972[35972]
+- Improve error reporting and fix IPv6 handling of TCP and UDP metric collection. {pull}[]
```

Comment:
This has been updated to include the PR number, but GitHub appears to be broken wrt rendering changes in PRs. See branch [here](https://github.com/efd6/beats/tree/s3573-udp-again).